Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/Follow.cs

```diff
@@ -0,0 +1,573 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Solvers
+{
+    /// <summary>
+    /// Follow solver positions an element relative in front of the forward axis of the reference.
+    /// The element can be loosly constrained (a.k.a. tag-along) so that it doesn't follow until it is too far.
+    /// </summary>
+    public class Follow : Solver
```

Comment:
I'm very curious (perhaps @keveleigh  you can explain since you understand the old solvers), isn't the Orbital solver supposed to implement tag-along behavior as well? Note that the name "follow" makes much more sense to me than "orbital" (which to me sounds like a planet orbiting me, so it's a behavior that orbits around as time goes on). But, I'd just like to understand once we get this in, why anybody would want to use Orbital.